packages/devtools_app/test/shared/span_parser_test.dart

```diff
@@ -196,12 +196,13 @@ String _buildGoldenText(String content, List<ScopeSpan> spans) {
   final buffer = StringBuffer();
   final spansByLine = groupBy(spans, (ScopeSpan s) => s.line! - 1);
 
-  final lines = content.split('\n');
+  final lines = content.trim().split('\n');
   for (var i = 0; i < lines.length; i++) {
     final line = lines[i];
     // We need the line length to wrap. If this isn't the last line, account for
     // the \n we split by.
-    final lineLength = line.length + (i == lines.length - 1 ? 0 : 1);
+    final newlineLength = i == lines.length - 1 ? 0 : 1;
```

Comment:
Nit: can we wrap `i == lines.length - 1` in parens for readability?

Reply:
Done!

Reply:
Also pushed another minor fix that was not accounting for spans that wrap that don't start at column 0 (in building golden text).